packages/components/organisms/f-checkout/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "0.88.0",
+  "version": "0.90.0",
```

Comment:
How come this has skipped a few versions? Are there PRs going in before yours? Might be worth adding a DNMY label if so.

Reply:
I was preempting those reviewed pr's (x2 ticks) before mine to be merged before mine